format_spec/metadata.md

```diff
@@ -17,12 +17,14 @@ my_array                            # array folder
 ```
 
 The metadata folder can contain any number of [timestamped](./timestamped_name.md):
-* [metadata files](#array-metadata-file)
+* [metadata files](#metadata-file)
 * [vacuum files](./vacuum_file.md)
 * **Note**: the timestamped names do _not_ include the format version.
 
 ## Metadata File
 
+Metadata files with the `.tmp` extension are used for local filesystems only, indicating the metadata is still being flushed to disk.
```

Comment:
Curious what others think of using a `.part` extension instead. Either way is fine by me so by all means we can stick with `.tmp`, just something that came to mind while reviewing this myself.

Reply:
`.tmp` has an unambiguous meaning, but you can add another extension before it if you want.

Also, this is an implementation detail that should not be mentioned in the spec. If we haven't already, we should state that readers should ignore files with names that don't follow the respective format.